dubbo-cluster/src/test/java/com/alibaba/dubbo/rpc/cluster/loadbalance/LoadBalanceTest.java

```diff
@@ -117,6 +127,18 @@ public void testRoundRobinLoadBalance_select() {
         }
     }
 
+    @Test
+    public void testConsistentHashLoadBalance_select() {
+        Map<Invoker, AtomicLong> counter = getInvokeCounter(ConsistentHashLoadBalance.NAME, invocationList);
+
+        long runs = invocationList.size();
+        for (Invoker minvoker : counter.keySet()) {
+            Long count = counter.get(minvoker).get();
+            float ratio = count/(runs / (0f + invokers.size()));
+            Assert.assertTrue("invoker ratio must be gt 0.5 and lt 2",ratio>0.5 && ratio< 2);
```

Comment:
Why `gt 0.5 and lt 2` is a reasonable ratio, can you explain?

Reply:
I also have the same concern.